.github/ISSUE_TEMPLATE/icon-request.md

```diff
@@ -5,21 +5,26 @@ title: '{icon metaphor} icon request'
 labels: 'visual design,triage,icon request'
 ---
 
-<!-- Ensure the title can be understood without the parent item's context, e.g. "Save button icon request" rather than just "new icon" -->
+<!-- Ensure the title can be understood without the parent item's context, by replacing "{icon metaphor}". E.g. "Save button icon request" -->
 
 # 🎨 Icon Request
 
 ## 📌 Request Background
 
 <!-- A short description of the request and requester. E.g. Who is the client and how do they expect to use the icon? -->
+<!-- Is the client using the icon with other Nimble components? When is the icon needed? -->
 
 ## 🧪 Research
 
-<!-- Enumerate the research you've done to identify the correct metaphor and find existing best practices. -->
-<!-- Include examples from [Font Awesome](https://fontawesome.com/search), [Noun Project](https://thenounproject.com), and [IconFinder](https://www.iconfinder.com/search) -->
+<!-- For each icon, include the following: -->
+<!-- Include examples from [Font Awesome](https://fontawesome.com/search), [Noun Project](https://thenounproject.com), or [IconFinder](https://www.iconfinder.com/search) -->
+<!-- Describe the research you've done to identify the correct icon metaphor and find existing best practices. -->
+<!-- Propose a name using the [icon naming guidance](https://github.com/ni/nimble/blob/main/packages/nimble-tokens/CONTRIBUTING.md#icon-naming) -->
+<!-- List icon metaphors. What will this icon mean in context? E.g. `nimble-icon-circle-check` represents _acknowledged status_. -->
 
 ## 🥅 Acceptance Criteria
 
+- [ ] Icon [name](https://github.com/ni/nimble/blob/main/packages/nimble-tokens/CONTRIBUTING.md#icon-naming) approved by PO and owners
```

Comment:
Do we have a process in mind for this? Do it at the triage meeting? Fred/Brandon writes a proposal in the issue and Jesse/Milan sign off on it?

Reply:
The triage meeting could work well - and if we need to re-add the triage tag after the visual design changes, that should be fine. Worst case, the naming is reviewed at PR time.